containers/mit-scheme/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# [Choice] MIT-Scheme version: 11.1
+ARG VARIANT=11.1
+FROM mcr.microsoft.com/vscode/devcontainers/mit-scheme:${VARIANT}
```

Comment:
We don't pre-build images from community contributions so this won't work/is unnecessary.

Reply:
Removed the Dockerfile